spec/models/queen_spec.rb

```diff
@@ -4,4 +4,46 @@
   it "should have a valid factory" do
     expect(create(:queen)).to be_valid
   end
+
+  describe "valid_move?" do
+    
+    let(:game) { create(:game)}
+    let(:queen) { create(:queen, x_pos:5, y_pos:5, game: game) }
```

Comment:
this might be fine because then in `should be false if queen is obstructed` we would need to reference the `game` record as: `queen.game...` instead of just `game`.